### PR TITLE
Fix broken build/configuration links

### PR DESCRIPTION
**Pull request by sbesson**
Documentation PR corresponding to changes in https://github.com/openmicroscopy/openmicroscopy/pull/4093 and https://github.com/openmicroscopy/openmicroscopy/pull/4237
- reviews all references to `local.properties.example`
- reviews all broken links to configuration files
- removes `admin ports` from the documentation and the autogenerated script
- describes usage of `omero.ports.prefix` for multiple ports deployment

This PR should turn https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/ green again.

### CODE REVIEW

File: omero/sysadmins/server-advanced-configuration.txt [hunk 1]
```diff
@@ -7,15 +7,14 @@ In some cases, the configuration properties will not suffice to fully
...
+:file:`etc/grid/windefault.xml` is used on
 Windows systems and both make use of
-:source:`templates.xml <etc/grid/templates.xml>`.
+:file:`etc/grid/templates.xml`.
 
 The default OMERO application descriptor deploys multiple server instances
 (Blitz-0, FileServer, Indexer-0, PixelData-0, ...) on
-a single node. Each server instance is defined by a ``server-template`` element in
-:source:`templates.xml <etc/grid/templates.xml>` with is own parameters.
+a single node. Each server instance is defined by a ``server-template`` element in :file:`etc/grid/templates.xml` with is own parameters.
```
Comment (mtbc) [inline omero/sysadmins/server-advanced-configuration.txt]: rather long line

Comment (mtbc) [inline omero/sysadmins/server-advanced-configuration.txt]: "with *its* own parameters"

File: omero/sysadmins/server-advanced-configuration.txt [hunk 2]
```diff
@@ -123,30 +122,36 @@ to do so directly in the application descriptor XML.
...
-section of the :doc:`/sysadmins/server-security` page for more information.
-
 By modifying the default OMERO ports, it is possible to run multiple
-OMERO servers on the same physical machine.
+OMERO servers on the same physical machine. All port numbers can be adjusted
+using the relevant :ref:`configuration properties <ports_configuration>`.
 
-::
+To run multiple servers on a single host, the easiest approach is prefix all
+ports (SSL, TCP, registry) using :property:`omero.ports.prefix`::
```
Comment (mtbc) [inline omero/sysadmins/server-advanced-configuration.txt]: "is *to* prefix"

**Comment by sbesson** [comment]
Pushed

File: omero/sysadmins/server-security.txt [hunk 3]
```diff
@@ -181,15 +181,13 @@ the clear is your session id.
...
 Administrators can configure OMERO such that unencrypted connections are
 not allowed, and the user's choice will be silently ignored. The |SSL| and
-non-SSL ports are configured in the etc/grid/default.xml and
-windefault.xml files, and as described above, default to 4064 and 4063
-respectively, and can be modified with command:
+non-SSL ports are configured in the :file:`etc/grid/default.xml` and
+:file:`etc/grid/windefault.xml` files, and as described above, default to 4064
+and 4063 respectively, and can be modified using the
+:ref:`ports_configuration` configuration properties. For instance, to prefix
+all ports by 1, use :property:`omero.ports.prefix`::
```
Comment (mtbc) [inline omero/sysadmins/server-security.txt]: maybe: to prefix all ports with '1',

**Comment by hflynn** [comment]
Looks fixed to me and the build is green again, good to merge @mtbc ?

**Comment by mtbc** [comment]
I indeed can live without https://github.com/openmicroscopy/ome-documentation/pull/1291#discussion_r41134728 being fixed if you want to merge!

**Comment by sbesson** [comment]
Will push in a sec

**Comment by hflynn** [comment]
Thanks, sorry I missed that

**Comment by sbesson** [comment]
Pushed.

**Comment by mtbc** [comment]
Certainly can live without quoting the "1" in any way. :smiley: Good to merge once Travis is green.

**Comment by hflynn** [comment]
--no-rebase

**Pull request merged**